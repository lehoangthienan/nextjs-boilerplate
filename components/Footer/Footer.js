import React, { Component } from 'react';
import LazyLoad from 'react-lazyload';

export default class Footer extends Component {
  render() {
    return (
      <LazyLoad height={400}>
        <div id="footer-desktop-main" className="row mt-2">
        </div>
      </LazyLoad>
    );
  }
}
