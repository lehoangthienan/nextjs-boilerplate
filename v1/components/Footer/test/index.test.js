import React from 'react'
import renderer from 'react-test-renderer'

import Footer from '../index'

describe('<Content />', () => {
  it('should render and match the snapshot', () => {
    const renderedComponent = renderer
      .create(
        <Footer />,
      )
      .toJSON()

    expect(renderedComponent).toMatchSnapshot()
  })
})
