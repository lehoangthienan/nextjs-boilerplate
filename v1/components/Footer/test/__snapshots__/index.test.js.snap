// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Content /> should render and match the snapshot 1`] = `
<div
  className="lazyload-placeholder"
  style={
    Object {
      "height": 400,
    }
  }
/>
`;
